src/api/app/controllers/message_controller.rb

```diff
@@ -17,12 +17,12 @@ def check_project_and_package
   end
 
   def list
-    if @package
-      @messages = @package.messages
+    @messages = if @package
```

Comment:
@Ana06, rubocop told me to align elsif with @messages, if you prefer I can put it like:
``` ruby
@message =
  if @package
    ...
  elsif @project
    ...
  end
```